app/controllers/application_controller.rb

```diff
@@ -77,6 +78,13 @@ def current_session
     @current_session ||= SessionActivation.find_by(session_id: cookies.signed['_session_id'])
   end
 
+  def current_theme
+    if Themes.instance.names.include? current_account&.user&.setting_theme
+      return current_account&.user&.setting_theme
```

Comment:
I do not think it needs `&.` operator here. If they gets `nil`, it is a programming error and should fail rather than returning `nil`.